src/rebuild/initiator.c

```diff
@@ -831,11 +807,13 @@ rebuild_obj_ult(void *data)
 
 		iov.iov_len = size;
 
+		enum_arg.oid = arg->oid;
 		enum_arg.kds = kds;
 		enum_arg.kds_cap = KDS_NUM;
 		enum_arg.kds_len = num;
 		enum_arg.sgl = &sgl;
 		enum_arg.sgl_idx = 1;
+		enum_arg.chk_key2big = true;
 		rc = dss_enum_unpack(VOS_ITER_DKEY, &enum_arg,
 				     rebuild_one_queue_cb, arg);
```

Comment:
The rebuild fetch/update needs be revised to fetch with stable view epoch instead of the iod epoch returned by enumeration, right?

Reply:
iteration will iterate the key/record by stable view, since the key/record will not change under stable view anyway, so we can use the returned epoch here?

Reply:
The records will be changed under stable view, 'stable view' means the 'view' is stable, not physical records.

Reply:
indeed, I should use stable view here